services/user.js

```diff
@@ -0,0 +1,23 @@
+(function () {
+	class UserService {
+		constructor(api) {
+			this.api = api;
+		}
+
+		getUser(userId) {
+			return new Promise((resolve, reject) => {
```

Comment:
Returning promises is not consistent with any service we already have, since we're actually returning callbacks in every service